wurst/systems/Knockback3.wurst

```diff
@@ -0,0 +1,225 @@
+/** By default, this package allows units to collide with destructables and
+    destroy them. To disable this feature, make the global
+    destroyDestructableSpeedThreshold a very high value:
+
+       Knockback3.destroyDestructableSpeedThreshold = 999999999.
```

Comment:
Then it needs to be configurable (with dependency management u wont be able to modify stdlib)